### DISCUSSION TITLE
不同语种在语音评估时候的差别显示

### DISCUSSION THREAD

**Discussion by lishan256** [ZuodaoTech/everyone-can-use-english#733] [General]
![1719548539949](https://github.com/xiaolai/everyone-can-use-english/assets/89031156/a7984e55-d405-49cd-aacd-c127f4a74d90)
![1719548332597](https://github.com/xiaolai/everyone-can-use-english/assets/89031156/08b7179c-87f2-4a77-a05a-e90d0c0cfbab)

您好，我做英语评估的时候，发现在每个单词上面都有每个字母发音的估分，能很清楚的看到哪个词的哪个字母发音如何，但是在做法语评估的时候，就没有。请问可以为法语也添加上这个功能吗？

**Comment by an-lee** [comment]
我检查一下看。
